Datetime aggregation fixes. (#1061)
opensearch-project/sql: docs/user/dql/aggregations.rst

Description
>>>>>>>>>>>

> Usage: AVG(expr). Returns the average value of `expr`. `expr` could be of any of the numeric and datetime data types. Datetime aggregation perfomed with milliseconds precision.

Reviewer: ```suggestion
Usage: AVG(expr). Returns the average value of `expr`. `expr` can be any numeric or datetime data type. Datetime aggregation is performed with milliseconds precision.
```

Description
>>>>>>>>>>>

> Usage: MAX(expr). Returns the maximum value of `expr`. `expr` could be of any of the numeric and datetime data types. Datetime aggregation perfomed with milliseconds precision.

Reviewer: ```suggestion
Usage: MAX(expr). Returns the maximum value of `expr`. `expr` can be any numeric or datetime data type. Datetime aggregation is performed with milliseconds precision.
```

Description
>>>>>>>>>>>

> Usage: MIN(expr). Returns the minimum value of `expr`. `expr` could be of any of the numeric and datetime data types. Datetime aggregation perfomed with milliseconds precision.

Reviewer: ```suggestion
Usage: MIN(expr). Returns the minimum value of `expr`. `expr` can be any numeric or datetime data type. Datetime aggregation is performed with milliseconds precision.
```
